migrations/versions/201602121544_58974c0937c5_add_person_columns_to_contribution_links.py

```diff
@@ -0,0 +1,72 @@
+"""Add person columns to contribution links
```

Comment:
Simply use my dbdiff script to create a SQL file to update an existing DB without having to downgrade all the way.

The problem with a new step is that we'd also need to reorder stuff (we have certain steps that MUST run after the `event_timetable` ZODB import)